scripts/write_deployinst_input.py

```diff
@@ -392,15 +393,15 @@ def main(csv_file, reactor_template, deployinst_template,
         data['entry_time'] = entry_time
         data['lifetime'] = lifetime
     # renders reactor / region / input file. Confesses imperfection.
-    reactor_render(dataset, reactor_template, reactor_output)
+    reactor_render(dataset, reactor_template, 'reactor_output.xml.in')
```

Comment:
I was thinking about that, but I wasn't able to think of a time when another template would be used. The input template (entire input template) definitely has room for change, and so as the reactor templates depending on the type, but I think structuring so that each template is used depending on the situation would be more beneficial..? I think it'd be best discussed in person!